skillful_network_server/src/main/java/fr/uca/cdr/skillful_network/repositories/user/FollowStateTrackerRepository.java

```diff
@@ -12,6 +12,6 @@
 public interface FollowStateTrackerRepository extends JpaRepository<FollowStateTracker, Long> {
     List<FollowStateTracker> findAllByFollower(User follower);
 //    boolean existByFollowerAndFollowed(User follower, User followed);
-    List<FollowStateTracker> findAllByFollowerAndFollowed(User follower, User followed);
+    FollowStateTracker findAllByFollowerAndFollowed(User follower, User followed);
```

Comment:
`findAll` qui retourne un élément simple ? On ne devrait pas alors l'appelle `findByFollowerAndFollowed` ?

Reply:
good finding !
new commit is coming as well